components/Grid/Grid.tsx

```diff
@@ -48,7 +48,7 @@ const GridItem = styled.div<{ hasInputFields: boolean; gridSize: number }>`
 	border-bottom: 1px solid ${(props) => props.theme.colors.borders};
 	border-right: 1px solid ${(props) => props.theme.colors.borders};
 	padding: 20px;
-	height: ${(props) => (props.hasInputFields ? '180px' : '135px')};
```

Comment:
this looks terrible when its always at 180px tbh. Let's just make the last row a bit longer

Reply:
or just make the font 1px smaller in that specific box.

Reply:
I would say no more than 150px is the highest we want to go for boxes without input fields